### ISSUE TITLE
Create Dockerfile

### ISSUE THREAD

**Issue by nesl247** [ConfigHubPub/Database-Manager#1]
When deploying ConfigHub to kubernetes, we want to be able to run this as a job before the deployment. We would rather not have to run this manually.

**Comment by grimwm** [comment]
Agreed.  I would like this to be a prebuilt jar or, preferably for the use-case we have (building confighub locally and bringing up local confighub dev instances for bug fixing it), move all of this code to the confighub project, allowing docker to easily get to it without additional manual steps.

**Comment by Jonakss** [comment]
I've made a pull request with some ideas. #9

**Comment by Jonakss** [comment]
It can be used with this image https://hub.docker.com/r/jonakss/confighub_dbm
